### DISCUSSION TITLE
What fields do we want in jobs?

### DISCUSSION THREAD

**Discussion by jmkoni** [latinxintech/job-board#6] [Ideas]
From: https://github.com/latinxintech/job-board/issues/5#issuecomment-764719814

- Do we want to have a field for min/max years of experience?
- Some sort of field to make it easy to find visa requirements
- Fully remote/remote friendly/in office
- Field to show if open to people with felony convictions
- What about allowing dreamers/undocumented?

**Comment by camilopayan** [comment]
> What about allowing dreamers/undocumented?

Based on the discussion in TQ-meta, something like "contractors/1099 can apply" is the way to do it

**Comment by oscar-edomex** [comment]
I appreciate the field for being open to people with past felony convictions. 

I agree with Camilo that for fully undocu peers a good way to welcome their application is to have "Open to Contractors / 1099"

For DACA / TPS (Work Authorization via Employment Authorization) it could be a field that says "Open to All Valid Work Authorization"

For Visas, it would be helpful for people to know if the company can offer sponsorship or not (H1B, etc). A lot of the time I've seen it as  "no employment sponsorship offered." This is important many people on student visas. 

I do want to mention that somebody with DACA/TPS does not require future employment sponsorship. Sometimes this is confused both amongst people with DACA and by employers. It doesn't mean that they wouldn't benefit from a potential sponsorship but the employers questions alludes more to whether you will need employment sponsorship in the future to continue working with them. Which, in this scenario is technically "no."
